pkgs/by-name/lo/local-content-share/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "local-content-share";
+  version = "31";
+
+  src = fetchFromGitHub {
+    owner = "Tanq16";
+    repo = pname;
+    rev = "v${version}";
```

```suggestion
    tag = "v${finalAttrs.version}";
```

`tag` is usually preferred over rev where applicable because it prevents accidentally fetching a branch with the same name